sdk/containerregistry/container-registry/src/containerRegistryClient.ts

```diff
@@ -63,9 +73,11 @@ export class ContainerRegistryClient {
     credential: TokenCredential,
     options: ContainerRegistryClientOptions = {}
   ) {
-    this.endpoint = endpointUrl;
-    this.credential = credential;
-    this.clientOptions = options;
+    this.registryUrl = endpointUrl;
+    const parsedUrl = new URL(endpointUrl);
```

Comment:
This is never supposed to be used on browsers. Is that correct? Just to confirm.

Reply:
There are other instances of URL in this, of course.

Reply:
`URL` is in all browsers we care about and once we depend on Node 10+ we won't even need the specialized import, since it will be global in Node as well

Reply:
yes, once we are there my copy of url*.ts can be removed.